lib/Lolifiers/GeekTime.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Monolol\Lolifiers;
+
+use Monolol\Lolifier;
+
+class GeekTime implements Lolifier
+{
+    const
+        MIN_INTERVAL = 1,
+        MAX_INTERVAL = 5;
```

Comment:
Sorry, I didn't see it during previous code review. These constants have to be removed